compiler/rustc_feature/src/removed.rs

```diff
@@ -139,7 +139,7 @@ declare_features! (
     /// Allows using `#[on_unimplemented(..)]` on traits.
     /// (Moved to `rustc_attrs`.)
     (removed, on_unimplemented, "1.40.0", None, None, None),
-    /// A way to temporarily opt out of opt in copy. This will *never* be accepted.
+    /// A way to temporarily opt out of opt-in copy. This will *never* be accepted.
```

Comment:
Here's a case with both a verb phrase (`opt out`) and a noun (`opt-in`) where the noun has compounded/hyphenated (`-`).